testament/categories.nim

```diff
@@ -101,7 +101,7 @@ proc flagTests(r: var TResults, cat: Category, options: string) =
   testSpec r, makeTest(filename, genopts, cat)
 
   when defined(windows):
-    testExec r, makeTest(filename, " cmd /c cd " & nimcache &
+    testExec r, makeTest(filename, " cd " & nimcache &
```

Comment:
Out of curiousity, why is this needed?

Reply:
in this particular case `cmd /c` would already be added by this PR's logic so I'm removing it from here; I am trying to fix CI failures; help welcome btw
* see https://dev.azure.com/nim-lang/255dfe86-e590-40bb-a8a2-3c0295ebdeb1/_apis/build/builds/2606/logs/68
* and https://dev.azure.com/nim-lang/255dfe86-e590-40bb-a8a2-3c0295ebdeb1/_apis/build/builds/2606/logs/72

Reply:
@timotheecour Sure, hop on gitter?